WooCommerce/Classes/ViewRelated/Orders/FulfillViewController.swift

```diff
@@ -244,14 +270,16 @@ private extension FulfillViewController {
 
     /// Setup a given UITableViewCell instance to actually display the specified Row's Payload.
     ///
-    func setup(cell: UITableViewCell, for row: Row) {
+    func setup(cell: UITableViewCell, for row: Row, at: IndexPath) {
         switch row {
         case .product(let item):
             setupProductCell(cell, with: item)
         case .note(let text):
             setupNoteCell(cell, with: text)
         case .address(let shipping):
             setupAddressCell(cell, with: shipping)
+        case .tracking:
+            setupTrackingCell(cell, at: at)
```

Comment:
Style nitpick. `setupTracking(cell, at: indexPath)` (I know, the rest of them should not say "Cell" as part of the method names either 😄)